src/AmberStatus.php

```diff
@@ -99,17 +99,20 @@ public function get_summary($url, $preferred_providers ) {
    */
   public function save_check(array $data) {
     $prefix = $this->table_prefix;
-
-    foreach (array('last_checked', 'next_check', 'status', 'url') as $key) {
+    foreach (array('url') as $key) {
```

Comment:
This fix has been taken from wordpress branch to common.
This will facilitate feature to display URLs which were not saved due to size issue. i.e. No Up/Down status in dashboard.